package/cfg/k3s-cis-1.8-hardened/node.yaml

```diff
@@ -151,19 +161,20 @@ groups:
                 op: eq
                 value: false
         remediation: |
-          If using a Kubelet config file, edit the file to set `authentication: anonymous: enabled` to
-          `false`.
-          If using executable arguments, edit the kubelet service file
-          $kubeletsvc on each worker node and
-          set the below parameter in KUBELET_SYSTEM_PODS_ARGS variable.
-          `--anonymous-auth=false`
-          Based on your system, restart the kubelet service. For example,
+          By default, K3s sets the --anonymous-auth to false. If you have set this to a different value, you
+          should set it back to false. If using the K3s config file /etc/rancher/k3s/config.yaml, remove any lines similar to below.
```

Comment:
```suggestion
          By default, K3s sets the --anonymous-auth to false. If you have set this flag to a different value or not at all, you should set it back to false. 
          If using the K3s config file /etc/rancher/k3s/config.yaml, remove any lines similar to below.
```

Reply:
If they set it not at all, that is the correct behavior. K3s will always set `--anonymous-auth=false`, which is what we want. The only scenario this check fails is if the user specifically sets it to true in their config, overriding K3s's defaults.

Reply:
Got it, I'll discard then.